config/m4/ucs.m4

```diff
@@ -150,6 +150,36 @@ CHECK_CROSS_COMP([AC_LANG_SOURCE([static int rc = 1;
                              supports __attribute__((constructor))])]
                 )
 
+
+#
+# Manual configuration of cacheline size
+#
+
+AC_ARG_WITH([cache-line-size],
+        [AC_HELP_STRING([--with-cache-line-size=SIZE],
+            [Build UCX with cache line size defined by user. This parameter
+             overwrites defaulf cache line sizes defines in
+             UCX (x86-64: 64, Power: 128, ARMv8: 64/128). The supported values are: 64, 128])],
+        [],
+        [with_cache_line_size=no])
+
+AS_IF([test "x$with_cache_line_size" != xno],[
+	     case ${with_cache_line_size} in
+                 64)
```

Comment:
I was thinking about this. Are you aware about any other system that is not 32 or 64 ? I wanted to add some level of protection that checks that the value is reasonable.